src/Random.java

```diff
@@ -0,0 +1,192 @@
+import java.io.File;
+import java.io.IOException;
+
+public class Random {
+    private InputOutput io = new InputOutput();
```

Comment:
assuming `Random` is supposed to be your Model class, it should not interact with the user or the InputOutput class. The goal of a the model is to store and work with internal data - if it needs information from the user, that work of getting the information and creating the model should happen inside the Controller (App.java in your case)